repos/system_upgrade/common/actors/peseventsscanner/libraries/pes_events_scanner.py

```diff
@@ -481,11 +482,8 @@ def apply_transaction_configuration(source_pkgs):
 
 
 def remove_leapp_related_events(events):
-    leapp_pkgs = [
-        'leapp', 'leapp-deps', 'leapp-upgrade-el7toel8', 'leapp-upgrade-el8toel9',
-        'leapp-upgrade-el7toel8-deps', 'leapp-upgrade-el8toel9-deps', 'python2-leapp',
-        'python3-leapp', 'snactor'
-    ]
+    leapp_pkgs = rpms.get_leapp_dep_packages(
+            major_version=['7', '8']) + rpms.get_leapp_packages(major_version=['7', '8'])
```

Comment:
@fernflower  add here please TODO note that we should change 7, 8 major versions to source & target  and the get_leapp_packages needs to by extended by 9 & 10 then - we will do it when we start to work on 9 -> 10 upgrades. people will be then better aware we need to change that then.

Reply:
@fernflower  thanks!